apm-agent-plugins/apm-spring-webmvc/apm-spring-webmvc-spring5/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-spring-webmvc</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.37.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-spring-webmvc-spring5</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <animal.sniffer.skip>false</animal.sniffer.skip>
```

Comment:
[minor] I think it's enabled by default, and most plugins that need to disable opt-out, thus setting it to `false` is probably not needed.

Reply:
It was skipped via the parent-pom (`spring-webmc`), that's why it was reenabled here. I removed the setting from the parent and moved it to the leaf projects to make it less confusing.